### PR TITLE
Added "Start with code" template and the list of categories

### PR DESCRIPTION
**Pull request by Lory1990**
## Description
I added a template to create a mia platform starter from code and added the list of marketplace categories 

## Pull Request Type

- [X] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [X] No sensitive content has been committed

### CODE REVIEW

File: docs/marketplace/add_to_marketplace/contributing_overview.md [hunk 1]
```diff
@@ -445,6 +445,39 @@ Here are listed some useful advice to strengthen your items:
 - **Test**: each service must have well-tested code;
 - **Logs**: each service should display the logs, to inform users about the actions they are currently performing and if any errors have been found during their execution.
 
+## Tips & Tricks
+
+ - **Where to host the zip with the code**: If you code your template on github you can access the zip file using the url https://github.com/&lt;username&gt;/&lt;project-name&gt;/archive/&lt;branch&gt;.tar.gz
+ - **.mia-platform folder**: This folder is ignored withn creating the repository, use it to store the images and the marketplace item configuration
```
Comment (fredmaggiowski) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:451]: Actually the directory is not ignored but used to provide specific files that are used to overwrite those in the main directory (e.g. the README). Then it won't be commited to the final repository, if this is what you meant I'd try and rephrase like


```suggestion
 - **.mia-platform folder**: This folder is used to overwrite files in the main directory and won't be commited in the created repository; you can use it to provide a different README file to developers or to store the images and the marketplace item configuration)
```

File: docs/marketplace/add_to_marketplace/contributing_overview.md [hunk 2]
```diff
@@ -445,6 +445,39 @@ Here are listed some useful advice to strengthen your items:
...
+
+## Category List
+The category list is constantly updated, check with your reponsible the updated list
```
Comment (fredmaggiowski) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:454]:
```suggestion

The category list is constantly updated, check with your Mia-Platform referent for the updated list
```

**Review by fredmaggiowski — APPROVED** [review]
Thanks @Lory1990, really appreciated!

LGTM let's hear from @akelity or @pmarchini to have a second approval

File: docs/marketplace/add_to_marketplace/contributing_overview.md [hunk 3]
```diff
@@ -445,6 +445,40 @@ Here are listed some useful advice to strengthen your items:
 - **Test**: each service must have well-tested code;
 - **Logs**: each service should display the logs, to inform users about the actions they are currently performing and if any errors have been found during their execution.
 
+## Tips & Tricks
+
+ - **Where to host the zip with the code**: If you code your template on github you can access the zip file using the url https://github.com/&lt;username&gt;/&lt;project-name&gt;/archive/&lt;branch&gt;.tar.gz
+ - **.mia-platform folder**: This folder is used to overwrite files in the main directory and won't be commited in the created repository; you can use it to provide a different README file to developers or to store the images and the marketplace item configuration)
```
Comment (fredmaggiowski) [inline docs/marketplace/add_to_marketplace/contributing_overview.md:451]:
```suggestion
 - **.mia-platform folder**: This folder is used to overwrite files in the main directory and won't be committed in the created repository; you can use it to provide a different README file to developers or to store the images and the marketplace item configuration)
```

File: .cspell.json [hunk 4]
```diff
@@ -5,7 +5,6 @@
   "language": "en",
   // words - list of words to be always considered correct
   "words": [
-    "addonsecurity",
```
Comment (fredmaggiowski) [inline .cspell.json]: Being an id we could use backtick-formatting (which is ignored) rather than adding an exception on the word :)

Comment (Lory1990) [reply .cspell.json]: Correct :)

**Review by pmarchini — APPROVED** [review]
LGTM

**Review by akelity — APPROVED** [review]
LGTM

**Pull request merged**